### PR TITLE
Build rpmbuild with parallel lzma support

### PR DESCRIPTION
**Pull request by chouquette**
This PR replicates what's done for other distributions by enabling parallel compression for XZ.

This is expected to save around 10 minutes per SUSE job in datadog-agent pipelines, as they currently take ≃15 minutes instead of ≃5 for other distributions: 
image

### CODE REVIEW

File: suse-x64/Dockerfile [hunk 1]
```diff
@@ -165,25 +168,39 @@ RUN curl -sL -O "https://ftp.gnu.org/gnu/binutils/binutils-${BINUTILS_VERSION}.t
...
 ENV PATH="/usr/local/binutils/bin:$PATH"
 
+# xz needs gettext 0.19.6 or newer
+RUN curl -LO https://ftp.gnu.org/gnu/gettext/gettext-${GETTEXT_VERSION}.tar.xz \
+    && echo "${GETTEXT_SHA256}  gettext-${GETTEXT_VERSION}.tar.xz" | sha256sum --check \
+    && tar xf "gettext-${GETTEXT_VERSION}.tar.xz" \
+    && cd "gettext-${GETTEXT_VERSION}" \
+    && ./configure --prefix=/usr && make -j$(nproc) && make install \
+    && cd / \
+    && rm -rf "gettext-${GETTEXT_VERSION}"
```
Comment (FlorentClarret) [inline suse-x64/Dockerfile:178]: nit - I think you do not remove the `tar.gz` file, just the folder.

File: suse-x64/Dockerfile [hunk 2]
```diff
@@ -165,25 +168,39 @@ RUN curl -sL -O "https://ftp.gnu.org/gnu/binutils/binutils-${BINUTILS_VERSION}.t
...
     && echo "13711268181a6e201eb9d4eb06384a7fcc42dcc6c9057a62c53472cfc5ba44b5  rpm-4.14.3.tar.bz2" | sha256sum --check \
     && tar -xjf /tmp/rpm-4.14.3.tar.bz2 \
     && cd rpm-4.14.3/ \
     && ./configure --with-posixmutexes --with-external-db --without-lua CPPFLAGS="-I/usr/include/nspr4 -I/usr/include/nss3 -I/usr/include/db4"\
-    && make \
+    && make -j$(nproc) \
     && make install \
-    && ldconfig
+    && ldconfig \
+    && rm -rf /tmp/rpm-4.14.3
```
Comment (FlorentClarret) [inline suse-x64/Dockerfile:197]: Same here with the tar file?

**Review by FlorentClarret — APPROVED** [review]
LGTM, just left one question/comment

File: suse-x64/Dockerfile (hunk 1 above)
Comment (chouquette) [reply suse-x64/Dockerfile:178]: Good point

File: suse-x64/Dockerfile (hunk 2 above)
Comment (chouquette) [reply suse-x64/Dockerfile:197]: Also a good point

File: suse-x64/Dockerfile (hunk 1 above)
Comment (chouquette) [reply suse-x64/Dockerfile]: Done, and replicated to the debian dockerfile which I copy pasted from

**Comment by chouquette** [comment]
image
The results are as expected (the 2 top lines are packaging durations for SUSE x86_64 jobs)

**Pull request merged**